src/main/scala/shade/memcached/Configuration.scala

```diff
@@ -24,6 +24,8 @@ import net.spy.memcached.ops.OperationQueueFactory
  * @param operationTimeout  is the default operation timeout; When the limit is reached, the
  *                          Future responses finish with Failure(TimeoutException)
  *
+ * @param shouldOptimize    If true, low-level optimization is in effect.
```

Comment:
Took me a while to track down exactly what happens if this is true, but it seems at the end of the day this option gets propagated to [`the MemcachedConnection`](https://github.com/couchbase/spymemcached/blob/master/src/main/java/net/spy/memcached/MemcachedConnection.java#L132-L135), where, the effect is:

> If true, optimisation will collapse multiple sequential get ops.

Can you please add that that description into the parameter ScalaDoc ?